cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/images/AddLaunchPermissionsToImageAction.java

```diff
@@ -136,7 +136,8 @@
                     .withUserGroupsString(userGroupsString)
                     .build();
 
-            return new QueryApiExecutor().execute(inputs, imageInputs);
+            Map<String, String> queryMapResult = new QueryApiExecutor().execute(inputs, imageInputs);
```

Comment:
@mihaitusa we plan to expose more outputs for some operations so i made a proactive change. As for the second comment it is not a default version for all operations, it's a default value per operation, and it was alligned with the value that has been tested (tok from description)